lib/server.js

```diff
@@ -16,12 +16,17 @@ var P = require('bluebird')
 P.promisifyAll(Memcached.prototype)
 var Raven = require('raven')
 const utils = require('./utils')
+const dataflow = require('./dataflow')
 
 // Create and return a restify server instance
 // from the given config.
 
 module.exports = function createServer(config, log) {
 
+  if (config.dataflow.enabled) {
```

Comment:
I notice we test this flag twice. You could just make this condiation a nop internally in `lib/dataflow`, then this code wouldn't have to know anything about the internal config of that module.

Reply:
I like this suggestion.

Reply:
Made that update.